services/networking/loadbalancer/taxonomy.md

```diff
@@ -0,0 +1,67 @@
+| CCC-050344  | Zonal Isolation                                       | Operation can be restricted to communicate only between resources deployed within a specific availability zone or region.                                                                                                                                                                                                     |
+| CCC-050345  | Regional Load Balancing                               | Load balancers can be distributed across multiple zones in the same region.                                                                                                                                                                                                                                                   |
+| CCC-050346  | Cross-Regional Load Balancing                         | Load balancers can be distributed across multiple regions.                                                                                                                                                                                                                                                                    |
+| CCC-050347  | Flow stickiness                                       | Allows requests from the same client to be consistently routed to the same backend server during the duration of a session.                                                                                                                                                                                                   |
+| CCC-050348  | Automatic failover to healthy backends                | Detect and redirect traffic away from unhealthy or failed backend servers to healthy ones automatically.                                                                                                                                                                                                                      |
+| CCC-050349  | Configurable standby backends                         | Configure backend targets that are only routed to if primary targets become unhealthy.                                                                                                                                                                                                                                        |
+| CCC-050350  | Connection draining on failover and failback          | Allows existing TCP sessions to remain active for a configurable time period even after backend VMs become unhealthy.                                                                                                                                                                                                         |
```

Comment:
This is possibly already covered with 'configurable timeouts'?

Reply:
I think it is covered by both CCC-050322 and CCC-050326 and we can remove this